packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -114,19 +115,35 @@ export const useArbTokenBridge = (
 
   const { addPendingTransaction } = useTransactionHistory(walletAddress)
 
+  const { destinationAddress } = useDestinationAddressStore()
+
   const {
     eth: [, updateEthL1Balance],
     erc20: [, updateErc20L1Balance]
   } = useBalance({
     provider: l1.provider,
-    walletAddress
+    walletAddress: walletAddress
```

Comment:
please revert the change as this is redundant

Reply:
Done